projects/extension/src/background/index.ts

```diff
@@ -259,21 +259,16 @@ chrome.runtime.onConnect.addListener((port) => {
           try {
             chain.sendJsonRpc(msg.jsonRpcMessage)
           } catch (error) {
-            // As documented in the protocol, malformed JSON-RPC requests are silently ignored.
```

Comment:
This was a subtle hint that the behavior in this code matches the protocol, and so if you change the behavior it doesn't match the protocol anymore